src/mon/PGMap.cc

```diff
@@ -3243,8 +3243,8 @@ void PGMap::get_health_checks(
 	summary += " reporting legacy (not per-pool) BlueStore stats";
       } else if (asum.first == "BLUESTORE_DISK_SIZE_MISMATCH") {
 	summary += " have dangerous mismatch between BlueStore block device and free list sizes";
-      } else if (asum.first == "BLUESTORE_NO_PER_PG_OMAP") {
-	summary += " reporting legacy (not per-pg) BlueStore omap usage stats";
+      } else if (asum.first == "BLUESTORE_NO_PER_POOL_OMAP") {
```

Comment:
The intention was to distinguish per-pool omap naming scheme introduced in Octopus and per-pg one introduced in Pacific to support bulk omap removals...
Given that do we still want to undo the rename?

Reply:
Oh, i see.. I wasn't aware of this change.

In that case, i think we need to just adjust the docs to reference the new NO_PER_PG_OMAP name

Reply:
actually, i think it makes sense to split out into separate health warnings and related options and messages from bluestore.  IIUC the pacific change only makes it so that PG removal is faster--it doesn't change stats reporting, so there isn't much reason to push users to migrate.  But the per-pool thing does matter, so bluestore needs to surface a separate flag for that?